dissect/esedb/table.py

```diff
@@ -5,21 +5,22 @@
 from typing import TYPE_CHECKING, Any
 
 from dissect.esedb import compression
+from dissect.esedb.btree import BTree
 from dissect.esedb.c_esedb import (
     CODEPAGE,
     COLUMN_TYPE_MAP,
     SYSOBJ,
     ColumnType,
     JET_coltyp,
 )
-from dissect.esedb.cursor import Cursor
 from dissect.esedb.exceptions import NoNeighbourPageError
 from dissect.esedb.index import Index
 from dissect.esedb.record import Record
 
 if TYPE_CHECKING:
     from collections.abc import Iterator
 
+    from dissect.esedb.cursor import Cursor
     from dissect.esedb.esedb import EseDB
     from dissect.esedb.page import Page
 
```

Comment:
I couldn't make a suggestion. But can you add a link to `Page` class in `root_page`